src/app/app.component.ts

```diff
@@ -34,11 +34,14 @@ export class AppComponent implements OnInit {
         this.analytics.page(event.url);
         const user = this.storage.getUser();
         if (user.uuid) {
-          this.analytics.identify(user.uuid, {
-            role: user.role ? user.role : null,
+          const role = user.role ? user.role : null;
+          const userInfo = {
+            role: (role === 'admin' || 'inst_admin') ? 'author' : role,
```

Comment:
hi @shawnm0705 I think there is a mistake here, this logic is strange unless it's intended. This if-shorthand will always return true, and surely the "author" is always the result no matter what value the role has.

Reply:
thanks @trtshen it is fixed now

Reply:
thanks Shawn